python/mlc_llm/serve/server/popen_server.py

```diff
@@ -68,9 +69,13 @@ def start(self) -> None:
         while query_result is None and attempts < timeout:
             try:
                 query_result = requests.get(openai_v1_models_url, timeout=60)
+                if query_result.status_code != 200:
+                    query_result = None
+                    attempts += 1
+                    time.sleep(0.1)
             except:  # pylint: disable=bare-except
                 attempts += 1
-                time.sleep(1)
+                time.sleep(0.1)
```

Comment:
You will need to increase the timeout attempt limit if lowering the gap to 0.1, or otherwise it timeouts after 6 seconds.

Reply:
Make sense. FIxed